guides/source/action_cable_overview.md

```diff
@@ -343,25 +363,6 @@ Broadcastings are purely an online queue and time-dependent. If a consumer is
 not streaming (subscribed to a given channel), they'll not get the broadcast
 should they connect later.
 
-Broadcasts are called elsewhere in your Rails application:
-
-```ruby
-WebNotificationsChannel.broadcast_to(
-  current_user,
-  title: 'New things!',
-  body: 'All the news fit to print'
-)
-```
-
-The `WebNotificationsChannel.broadcast_to` call places a message in the current
-subscription adapter's pubsub queue under a separate broadcasting name for each user.
-The default pubsub queue for Action Cable is `redis` in production and `async` in development and
-test environments. For a user with an ID of 1, the broadcasting name would be `web_notifications:1`.
-
-The channel has been instructed to stream everything that arrives at
-`web_notifications:1` directly to the client by invoking the `received`
-callback.
-
```

Comment:
I removed this text because it is duplicated from the "Example 2: Receiving New Web Notifications" section further below, and because the "Streams" section above now explicitly introduces `broadcast` and `broadcast_to`.